From a7580995758305833aa269d7972df13040981473 Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Wed, 1 Jan 2020 10:44:51 +0000
Subject: [PATCH] <linux/limits.h> is implicitly included via <limits.h> but
 not required here

Source/GmmLib/Utility/GmmLog/GmmLog.cpp:36:10: fatal error: 'linux/limits.h' file not found
 #include <linux/limits.h>
          ^~~~~~~~~~~~~~~~
---
 Source/GmmLib/Utility/GmmLog/GmmLog.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Source/GmmLib/Utility/GmmLog/GmmLog.cpp b/Source/GmmLib/Utility/GmmLog/GmmLog.cpp
index 4ee045b6..e864fac6 100644
--- a/Source/GmmLib/Utility/GmmLog/GmmLog.cpp
+++ b/Source/GmmLib/Utility/GmmLog/GmmLog.cpp
@@ -34,7 +34,6 @@ OTHER DEALINGS IN THE SOFTWARE.
 #include <sys/stat.h>
 #include <unistd.h>
 #include <fstream>
-#include <linux/limits.h>
 #endif
 
 /// Logger instance shared by all of GmmLib within a process
